ruby-challenges.rb

```diff
@@ -31,3 +56,23 @@
 # Expected output: 'LEARN is not a palindrome'
 palindrome_tester3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
+
+def palindrome?(string)
+    if string == string.reverse
+      return 'is a palindrome'
+    else
+      puts 'is not a palindrome'
+    end
+ end
+p palindrome?(palindrome_tester1)
+p palindrome?(palindrome_tester2)
+p palindrome?(palindrome_tester3)
```

Comment:
Keep in mind that currently, your method is case-sensitive so the string and its reversed version is not exactly the same due to capital letters.